types/paypal__paypalhttp/tsconfig.json

```
@@ -0,0 +1,30 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
```

```suggestion
            "es6"
```
why do you need `dom` here?

---

I am using console in the tests but I will remove it.